contrib/pg_stat_statements/expected/pg_stat_statements.out

```diff
@@ -95,26 +95,27 @@ EXECUTE pgss_test(1);
 (1 row)
 
 DEALLOCATE pgss_test;
-SELECT query, calls, rows FROM pg_stat_statements ORDER BY query COLLATE "C";
-                       query                       | calls | rows 
----------------------------------------------------+-------+------
- PREPARE pgss_test (int) AS SELECT $1, $2 LIMIT $3 |     1 |    1
- SELECT $1                                        +|     4 |    4
-                                                  +|       | 
-   AS "text"                                       |       | 
- SELECT $1 + $2                                    |     2 |    2
- SELECT $1 + $2 + $3 AS "add"                      |     3 |    3
- SELECT $1 AS "float"                              |     1 |    1
- SELECT $1 AS "int"                                |     2 |    2
- SELECT $1 AS i UNION SELECT $2 ORDER BY i         |     1 |    2
- SELECT $1 || $2                                   |     1 |    1
- SELECT pg_stat_statements_reset()                 |     1 |    1
- WITH t(f) AS (                                   +|     1 |    2
-   VALUES ($1), ($2)                              +|       | 
- )                                                +|       | 
-   SELECT f FROM t ORDER BY f                      |       | 
- select $1::jsonb ? $2                             |     1 |    1
-(11 rows)
+SELECT count(*), query, sum(calls) calls, sum(rows) rows FROM pg_stat_statements GROUP BY query ORDER BY query COLLATE "C";
+ count |                              query                              | calls | rows 
+-------+-----------------------------------------------------------------+-------+------
+     1 | PREPARE pgss_test (int) AS SELECT $1, $2 LIMIT $3               |     1 |    1
+     1 | SELECT $1                                                      +|     4 |    4
+       |                                                                +|       | 
+       |   AS "text"                                                     |       | 
+     1 | SELECT $1 + $2                                                  |     2 |    2
+     1 | SELECT $1 + $2 + $3 AS "add"                                    |     3 |    3
+     1 | SELECT $1 AS "float"                                            |     1 |    1
+     1 | SELECT $1 AS "int"                                              |     2 |    2
+     1 | SELECT $1 AS i UNION SELECT $2 ORDER BY i                       |     1 |    2
+     1 | SELECT $1 || $2                                                 |     1 |    1
+     1 | SELECT pg_stat_statements_reset()                               |     1 |    1
+     3 | SELECT pg_stat_statements_reset_segments(userid, dbid, queryid) |     3 |    0
```

Comment:
I wonder why do we get a zero amount of rows for a current call? Cause in a case of the `SELECT pg_stat_statements_reset()` we got a single row in result.

Reply:
Executors run MotionSender node, which don't return any tuples, while dispatcher runs MotionReceiver